### PR TITLE
Mouse Tweaks, and fixes - Blame Polaris. <3

### PR DESCRIPTION
**Pull request by Rykka-Stormheart**
Several mice tweaks, mainly:
Uncommented out the mice pull_size and mob_pull - this fixes mice pulling lockers and shit.
Reduced mouse spawn time by half, since mice are nerfed now to not be able to pull large shit. (As they shouldn't be able to.)
Merged/incorporated file from [Mithrandalf's Pull Request - Mouse tweaks and bugfixes](https://github.com/VOREStation/VOREStation/pull/5709) - his was simple_animal, I moved it to simple_mob. All credits for changes in mouse_vr.dm go to them.

Small Fixes:
Added a little extra comment to the mice examine text. <3
Fixed the "Observe" Spawn timer - it originally said 5 minutes, fixed it to show the "actual" respawn time - 60 seconds.

### CODE REVIEW

File: code/global.dm [hunk 1]
```diff
@@ -53,7 +53,7 @@ var/list/lastsignalers = list() // Keeps last 100 signals here in format: "[src]
 var/list/lawchanges    = list() // Stores who uploaded laws to which silicon-based lifeform, and what the law was.
 var/list/reg_dna       = list()
 
-var/mouse_respawn_time = 5 // Amount of time that must pass between a player dying as a mouse and repawning as a mouse. In minutes.
+var/mouse_respawn_time = 2.5 // Amount of time that must pass between a player dying as a mouse and repawning as a mouse. In minutes. VOREStation Edit - Changed timer to 2.5, halved from 5.
```
Comment (SpadesNeil) [inline code/global.dm]: This var can be overridden in global_vr.dm I think.

File: code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm [hunk 2]
```diff
@@ -13,8 +13,8 @@
 
 	mob_size = MOB_MINISCULE
 	pass_flags = PASSTABLE
-//	can_pull_size = ITEMSIZE_TINY
-//	can_pull_mobs = MOB_PULL_NONE
+	can_pull_size = ITEMSIZE_TINY // Vorestation Edit - Rykka - Uncommented these. Not sure why they were commented out?
+	can_pull_mobs = MOB_PULL_NONE // Vorestation Edit - Rykka - Uncommented these. Not sure why they were commented out?
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm]: These can both be added under mouse_vr.dm and the original commented out bits can stay. Mouse_vr.dm will override this file with any changes that are made.

File: code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm [hunk 3]
```diff
@@ -52,7 +52,7 @@
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
 	icon_rest = "mouse_[body_color]_sleep"
-	desc = "A small [body_color] rodent, often seen hiding in maintenance areas and making a nuisance of itself."
+	desc = "A small [body_color] rodent, often seen hiding in maintenance areas and making a nuisance of itself. And stealing cheese, or annoying the chef. SQUEAK."
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm]: Again, mouse_vr.dm

File: code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse_vr.dm [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/mob/living/simple_mob/animal/passive/mouse
+	no_vore = 1 //Mice can't eat others due to the amount of bugs caused by it.
+	vore_taste = "cheese"
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse_vr.dm]: All those changes I just mentioned can slip in right under this line here.

File: code/modules/mob/new_player/new_player.dm [hunk 5]
```diff
@@ -114,7 +114,7 @@
 
 	if(href_list["observe"])
 
-		if(alert(src,"Are you sure you wish to observe? You will have to wait 5 minute before being able to respawn!","Player Setup","Yes","No") == "Yes") //Vorestation edit
+		if(alert(src,"Are you sure you wish to observe? You will have to wait 60 seconds before being able to respawn!","Player Setup","Yes","No") == "Yes") //Vorestation edit - Rykka corrected to 60 seconds to match current spawn time
```
Comment (SpadesNeil) [inline code/modules/mob/new_player/new_player.dm]: This one actually can't be changed in new_player_vr.dm so easily, so a vorestation edit here is fine.

**Comment by Mithrandalf** [comment]
Alright neat, I'll close mine.

File: code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm (hunk 2 above)
Comment (Rykka-Stormheart) [reply code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm]: Updooted <3

File: code/global_vr.dm [hunk 6]
```diff
@@ -35,4 +35,6 @@ var/global/list/acceptable_fruit_types= list(
 											"tomato",
 											"watermelon",
 											"wheat",
-											"whitebeet")
\ No newline at end of file
+											"whitebeet")
+
+mouse_respawn_time = 2.5 // Amount of time that must pass between a player dying as a mouse and repawning as a mouse. In minutes. VOREStation Edit - Changed timer to 2.5, halved from 5.
```
Comment (SpadesNeil) [inline code/global_vr.dm]: Oi put this at the top of the file, not the bottom. Also will this even compile? There's no object path.

Comment (Rykka-Stormheart) [reply code/global_vr.dm]: I'll get that done! Ascian said to remove the var/ - if Travis fails again after moving it I'll try the var/

Comment (Rykka-Stormheart) [reply code/global_vr.dm]: I'm getting this when moving that line to the top of the file.

`code\global.dm:56:error: mouse_respawn_time: duplicate definition`
`code\global_vr.dm:1:error: mouse_respawn_time: previous definition`

Granted, I tried var/mouse_respawn_time = - I'm going to remove the var/ and see if it successfully compiles again.

Comment (Rykka-Stormheart) [reply code/global_vr.dm]: Okay, with var/ removed, the error changes to this:

code\global_vr.dm:1:error: mouse_respawn_time: duplicate definition
code\global_vr.dm:1:error: 2.5: duplicate definition
code\global_vr.dm:1:error: =: instruction not allowed here

Should I be commenting out the original var/mouse_respawn_time in the Polaris files and using var/mouse_respawn_time in the _vr.dm?

Comment (SpadesNeil) [reply code/global_vr.dm]: I guess you might as well just go back to your original vorestation edit. Oh well.

**Comment by KasparoVy** [comment]
Mighty mouse :(

**Pull request merged**